cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -816,10 +819,33 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           p(
             """The resulting role should have privileges that allow executing all procedures with elevated privileges
               |except `db.labels` which is only allowed to execute using the users own privileges:""".stripMargin)
-          query("SHOW ROLE deniedBoostedProcedureExecutor4 PRIVILEGES", NoAssertions) {
+          query("SHOW ROLE deniedBoostedProcedureExecutor4 PRIVILEGES AS COMMANDS", NoAssertions) {
             p("Lists all privileges for role 'deniedBoostedProcedureExecutor4'")
             resultTable()
           }
+
+          p("Example 5: How would the privileges from example 1-4 affect a procedures output")
+          p(
+            """Lets pretend there exists a procedure called `myProc`.
+              |This procedure gives the result `A` and `B` for a user with `EXECUTE PROCEDURE` privilege
+              |and `A`, `B` and `C` for a user with `EXECUTE BOOSTED PROCEDURE` privilege.
+              |Now lets adapt the privileges in examples 1 to 4 to apply on this procedure and show what would be returned.""".stripMargin)
+          p(
+            """With the privileges from example 1, granted `EXECUTE PROCEDURE *` and denied `EXECUTE BOOSTED PROCEDURE myProc`,
+              |the `myProc` procedure would return the result `A` and `B`.""".stripMargin)
+          p(
+            """With the privileges from example 2, granted `EXECUTE BOOSTED PROCEDURE *` and denied `EXECUTE PROCEDURE myProc`,
+              |the `myProc` procedure would throw an error on no access to the procedure.""".stripMargin)
```

Comment:
```suggestion
              |execution of the `myProc` procedure would not be allowed.""".stripMargin)
```

Same update for the other places.